examples/angular/src/pages/ui/components/authenticator/sign-in-totp-mfa/sign-in-totp-mfa.component.html

```diff
@@ -1,4 +1,4 @@
-<amplify-authenticator>
+<amplify-authenticator [formFields]="formFields">
```

Comment:
Why is this necessary? Is this required for all customers using TOT{? Could it be considered a breaking change?

Reply:
This is the examples app. I added a new e2e test that now checks the setup TOTP QR field. So I had to add this new fields in the vue/angular/react e2e tests.